content/library_constants.tex

```diff
@@ -176,7 +164,6 @@
 \LibConstDecl{SHMEM\_MINOR\_VERSION}
 \begin{DeprecateBlock}
   \LibConstDecl{\_SHMEM\_MINOR\_VERSION}
-  \LibConstDecl[\Fortran]{SHMEM\_MINOR\_VERSION}
```

Comment:
Implementer's choice? If the implementation can provide C/C++ interfaces for 1.5 and Fortran interfaces for 1.4, they could set MAJOR/MINOR to 1.4 for Fortran and interoperate with the C/C++ 1.5 implementation. Or we can punt on this question if it becomes relevant for a different issue because the Fortran interfaces are wonky/broken anyway.

Reply:
I'm not sure I have a preference either.  In SOS, this is manually updated in each header, so we will need to make a choice whether to update it or not.  Interested to know if anyone has a preference.  🤷‍♂